components/multimedia/timidity++/Makefile

```diff
@@ -12,12 +12,14 @@
 # Copyright 2018 Harry Liebel <hliebel@gmail.com>
 #
 
-BUILD_BITS= 64
+
+# disable CONFIGURE_DEFAULT_DIRS, because timidity++ does not support --localdir
+CONFIGURE_DEFAULT_DIRS=no
```

Comment:
I assume it should read `--localedir` instead of `--localdir`.  If so, then please set `CONFIGURE_DEFAULT_DIRS` to `pre-2.60` to solve the issue.

Reply:
... for details please see #20412.